versions.json

```diff
@@ -10,5 +10,6 @@
   "kontrol": "1.0.53",
   "just": "1.34.0",
   "binary_signer": "1.0.4",
-  "semgrep": "1.90.0"
+  "semgrep": "1.90.0",
+  "asterisc": "cl/contract-updates"
```

Comment:
Why this branch and not master?

Reply:
https://github.com/ethereum-optimism/asterisc/pull/93 isn't merged yet.